projects/fsastorefront-e2e-cypress/cypress/helpers/my-account/myAccountPages.ts

```diff
@@ -82,3 +82,19 @@ export function checkOrderHistoryContent(price) {
       cy.get('.cx-order-history-total').contains(price);
     });
 }
+
+export function retrieveQuote(length, product) {
+  cy.selectOptionFromDropdown({
+    menuOption: 'My Account',
+    dropdownItem: 'Quotes & Applications',
+  });
+  cy.get('h2').should('contain.text', 'Quotes & Applications');
+  cy.get('.info-card').should('have.length', length);
+  cy.get('h6')
+    .contains(product)
+    .parent()
+    .within(() => {
```

Comment:
If I'm not mistaken, you are already on the same level as h6, so no need to go up to the parent and then go again inside to get the link Retrieve.

Reply:
Used siblings 
Thanks